x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_creation/common_components.cy.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ruleFields } from '../../../data/detection_engine';
+import { getTimeline } from '../../../objects/timeline';
+
+import { CUSTOM_RULES_BTN } from '../../../screens/alerts_detection_rules';
+import {
+  ABOUT_CONTINUE_BTN,
+  ABOUT_EDIT_BUTTON,
+  CUSTOM_QUERY_INPUT,
+  DEFINE_CONTINUE_BUTTON,
+  DEFINE_EDIT_BUTTON,
+  RULE_NAME_INPUT,
+  SCHEDULE_CONTINUE_BUTTON,
+} from '../../../screens/create_new_rule';
+import { createTimeline } from '../../../tasks/api_calls/timelines';
+import { deleteAlertsAndRules } from '../../../tasks/common';
+import {
+  createAndEnableRule,
+  expandAdvancedSettings,
+  fillCustomInvestigationFields,
+  fillDescription,
+  fillFalsePositiveExamples,
+  fillFrom,
+  fillNote,
+  fillReferenceUrls,
+  fillRiskScore,
+  fillRuleName,
+  fillRuleTags,
+  fillSeverity,
+  fillThreat,
+  fillThreatSubtechnique,
+  fillThreatTechnique,
+  importSavedQuery,
+} from '../../../tasks/create_new_rule';
+import { login, visit } from '../../../tasks/login';
+import { RULE_CREATION } from '../../../urls/navigation';
+
+// This test is meant to test touching all the common various components in rule creation
+// to ensure we don't miss any changes that maybe affect one of these more obscure UI components
+// in the creation form. For any rule type specific functionalities, please include
+// them in the relevant /rule_creation/[RULE_TYPE].cy.ts test.
+describe('Common rule creation components', { tags: ['@ess', '@serverless'] }, () => {
```

Comment:
Since this is not skipped, do we know for sure this test is not flaky in Serverless?